### PR TITLE
fix: Enable devenv completion to work without updating.

### PR DESCRIPTION
**Pull request by ChrisPRobinson**
[DTSS-429]

## What this PR does / why we need it
fix: Enable devenv completion to work without updating.

**JIRA ID**:DTSS-429

## Notes for your reviewers

[DTSS-429]: https://outreach-io.atlassian.net/browse/DTSS-429?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: cmd/devenv/devenv.go [hunk 1]
```diff
@@ -69,6 +69,14 @@ func main() {
 		`),
 		EnableBashCompletion: true,
 		Before: func(c *cli.Context) error {
+			// NOTE: You would think you can check the the c.Command.Name == "completion"
```
Comment (coding-paras) [inline cmd/devenv/devenv.go:72]: Wow. I wonder why does this doesn't work.

File: cmd/devenv/devenv.go [hunk 2]
```diff
@@ -69,6 +69,14 @@ func main() {
...
+			// in before to see if that command is being run, you would be wrong
+			// Using the args passed in to see if the completion command
+			// was provided. Other global flags are just ignored.
+			if c.Args().Len() > 0 && c.Args().First() == "completion" {
```
Comment (coding-paras) [inline cmd/devenv/devenv.go:76]: [nit] [c.Args().First()](https://github.com/urfave/cli/blob/master/args.go#L28) is nil safe i.e. you don't need to explicitly check `c.Args().Len() > 0`

**Review by coding-paras — APPROVED** [review]

**Review by george-e-shaw-iv — APPROVED** [review]
approved, but paras is right

File: cmd/devenv/devenv.go (hunk 2 above)
Comment (ChrisPRobinson) [reply cmd/devenv/devenv.go]: Ah, should have looked deeper that the impl.

**Comment by ChrisPRobinson** [comment]
Updated to fix what @coding-paras pointed out.

**Pull request merged**